go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/celestiaorg/go-square v1.1.1
 	github.com/celestiaorg/go-square/v2 v2.1.0
 	github.com/celestiaorg/knuu v0.16.2
-	github.com/celestiaorg/nmt v0.22.2
+	github.com/celestiaorg/nmt v0.23.0
```

Comment:
this was bumped in core and on main, so either we revert the bump in core or keep it in this PR...